content/bn/docs/reference/glossary/contributor.md

```diff
@@ -0,0 +1,18 @@
+---
+title: অবদানকারী
+id: contributor
+date: 2018-04-12
+full_link: 
+short_description: >
+  কুবেরনেটিস প্রকল্প বা কমিউনিটিকে সহায়তার জন্য যিনি কোড, ডকুমেন্টেশন, বা তাদের সময় দান করেন।
```

Comment:
@Noblesse013 

For `kubernetes` localization please use `কুবারনেটিস` instead of `কুবেরনেটিস`.